tests/python/pants_test/engine/test_isolated_process.py

```diff
@@ -260,7 +260,7 @@ def test_file_in_directory_creation(self):
 
   def test_not_executable(self):
     file_name = 'echo.sh'
-    file_contents = b'#!/bin/bash -eu\necho "Hello"\n'
+    file_contents = b'#!/bin/sh -eu\necho "Hello"\n'
```

Comment:
ditto.

Reply:
tests relying on the environment is weird. Also, this is POSIX compatible so rely on `/bin/sh` which is nearly always POSIX.